.styluaignore

```diff
@@ -1,11 +1,12 @@
-/build
-/runtime/lua/coxpcall.lua
-/runtime/lua/vim/_meta
-/runtime/lua/vim/re.lua
+# Note: `build/` doesn't work, use full glob (e.g. `build/**/*.lua`)
+build/**/*.lua
+runtime/lua/coxpcall.lua
+runtime/lua/vim/_meta/*.lua
+runtime/lua/vim/re.lua
 
-/test/functional
-/test/functional/fixtures/lua/syntax_error.lua
```

Comment:
I was thinking that removing the leading slash may be preferred since it seems more flexible. SO that change might still be worthwhile.

Reply:
Rebased after #26398 merged to master. It's your call to merge this workaround or just close it.

Reply:
> I was thinking that removing the leading slash may be preferred since it seems more flexible. SO that change might still be worthwhile.

Unsure what "flexible" means in this context. The one with the leading slash is simply more correct than the one without it (unless we want these rules to apply globally, which I cannot fathom why we would tbh). Stylua maintainer is very responsive and very quick with fixes, I don't think we need to resort to quick solutions.

Reply:
Yes, I agree. Cleaning up remaining leading slashes can be done as a minor fixup to #26398. Let me close this one.